src/screen/GameScreen.java

```diff
@@ -242,6 +242,17 @@ protected boolean update() {
 		super.update();
 
 		if (gameProgress) {
+			if (Globals.getInputManager().isDoubleTab(KeyEvent.VK_RIGHT) && doubleTapCooldown.getRemainingTime() == 0 ) {
```

Comment:
This is not handle ship's key setting.
And ship's movement is out of there class. This is should be in ship class